src/compiler/utilitiesPublic.ts

```diff
@@ -307,6 +307,7 @@ export function sortAndDeduplicateDiagnostics<T extends Diagnostic>(diagnostics:
 /** @internal */

```

Comment:
It would be helpful to add a NOTE comment 🙇 

https://github.com/petamoriken/TypeScript/blob/4b12d82b6bb4c8e2ad7df3219ae7282e360e62f9/src/compiler/commandLineParser.ts#L147-L149